container/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "metric_prefix": "container.",
+  "metric_to_check": "container.uptime",
+  "name": "container",
+  "short_description": "Track your container metrics with Datadog",
+  "guid": "39d40858-0d56-4623-bd1d-864790d0c894",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "windows"
+  ],
+  "public_title": "Datadog-Container Integration",
+  "categories": [
+    "containers"
+  ],
+  "type": "check",
+  "is_public": true,
```

Comment:
Just to confirm, should this be public yet?

Reply:
Yes it will be available from 7.33

Reply:
How about we wait to set this to `true` until 7.33 is released?

Reply:
Updated